content/en/synthetics/api_tests/dns_tests.md

```diff
@@ -0,0 +1,162 @@
+---
+title: DNS Tests
+kind: documentation
+description: Monitor resolvability and lookup times of your DNS records
+aliases:
+  - /synthetics/dns_test
+  - /synthetics/dns_check
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetic Monitoring"
+- link: "/getting_started/synthetics/api_test"
+  tag: "Documentation"
+  text: "Get started with API tests"
+- link: "/synthetics/private_locations"
+  tag: "Documentation"
+  text: "Test DNS resolution of your internal endpoints"
+- link: "https://www.datadoghq.com/blog/monitor-dns-with-datadog/"
+  tag: "Blog"
+  tewt: "DNS monitoring with Datadog"
+---
+
+## Overview
+
+DNS tests allow you to proactively monitor the **resolvability and lookup times of your DNS records using any nameserver**. If resolution is unexpectedly slow or if a DNS server answers with unexpected A, AAAA, CNAME, TXT, or MX entries, Datadog sends you an **alert with details on the failure**, allowing you to quickly pinpoint the root cause of the issue and fix it.
+DNS tests can run from [managed][1] and [private locations][2] allowing you to run tests on **public domains** as well as **internal services** that might be using internal DNS.
+
+## Configuration
+
+After choosing the type of test you are willing to create ([`HTTP`][3], [`SSL`][4], [`TCP`][5], or [`DNS` test][6]), you can define your test's request.
+
+### Define request
+
+{{< img src="synthetics/api_tests/dns_test_config.png" alt="Define DNS query"  style="width:90%;" >}}
+
+1. Specify the **Domain** you want your test to query, e.g., `www.example.com`.
+2. Specify the `DNS Server` to use (optional). If not specified, your DNS test performs resolution using `8.8.8.8`, with a fallback on `1.1.1.1` and an internal AWS DNS server.
```

Comment:
```suggestion
2. Specify the `DNS Server` to use (optional), it can be a domain name or an IP address. If not specified, your DNS test performs resolution using `8.8.8.8`, with a fallback on `1.1.1.1` and an internal AWS DNS server.
```